TestCases/flamelet/05_laminar_premixed_ch4_flame_cfd_axi/lam_prem_ch4_cfd_axi.cfg

```diff
@@ -113,7 +115,7 @@ CONV_RESIDUAL_MINVAL= -15
 CONV_STARTITER= 10
 CONV_CAUCHY_ELEMS= 100
 CONV_CAUCHY_EPS= 1E-6
-SCREEN_OUTPUT= INNER_ITER WALL_TIME RMS_VELOCITY-X RMS_VELOCITY-Y RMS_PRESSURE RMS_PROGRESS_VARIABLE RMS_TOTAL_ENTHALPY
+SCREEN_OUTPUT= INNER_ITER WALL_TIME RMS_VELOCITY-X RMS_VELOCITY-Y RMS_PRESSURE RMS_ProgressVariable RMS_EnthalpyTot
```

Comment:
Should we remain consistent with upper case letters?

Reply:
The reason why lower-case letters are used here is because the RMS options for the controlling variables have to use the same names as those specified under `CONTROLLING_VARIABLE_NAMES` in the config file. In my opinion, it would be more confusing if you as the user would have to specify the screen and history output RMS options in upper case, even though the controlling variable names specified are in lower case.